test/integration/README.md

```diff
@@ -6,22 +6,6 @@ This package contains a test environment and suite for our payment flows and api
 
 The test environment consists of a docker network that includes the `backend` and `auth` services and a shared database for two mock account servicing entities, `cloud-nine-wallet-test` and `happy-life-bank-test`. The test suite is run from the host machine and performs the mock accounting logic and integration server for each of these entities. The tests rely on `mock-account-service-lib` to perfom seeding and business logic in a consistent manner with `mock-account-servicing-entity`.
 
-# Running the tests
```

Comment:
It's the hostfile setup which is no longer necessary. After removing that I thought the rest (`pnpm install` and `pnpm --filter integration run-tests`) was just kinda self evident. I can return that section with those commands if you think it would still be helpful.

Reply:
Maybe we can add both 

`pnpm --filter integration build:deps` and
`pnpm --filter integration run-tests` 

since those are necessary to run the tests locally?

Reply:
`pnpm --filter integration build:deps` is handled by the `run-tests.sh` script.

I think the `package.json` scripts are pretty self-documenting but I can return instructions to run tests with `pnpm --filter integration run-tests` here with some details on what the run-tests.sh script is doing.

Reply:
updated the readme as described : [ac91c46](https://github.com/interledger/rafiki/pull/2654/commits/ac91c469fd42237dc07c55d2c77d2a105d3c93f6)